test/TestExpectations.json

```diff
@@ -911,6 +911,12 @@
     "parameters": ["webDriverBiDi"],
     "expectations": ["FAIL"]
   },
+  {
+    "testIdPattern": "[oopif.spec] OOPIF should support OOP iframes getting detached",
```

Comment:
I think it would be easier to just flip FF expectations that were marked FAIL to be PASS